### PR TITLE
Maistra 771: default elasticsearch so template can render

### PR DESCRIPTION
**Pull request by knrc**
Work in progress as it needs MAISTRA-706 PR to merge first

### CODE REVIEW

**Review by dgn — APPROVED** [review]

**Comment by knrc** [comment]
rebased on latest maistra-1.0

File: tmp/build/patch-jaeger.sh [hunk 1]
```diff
@@ -5,6 +5,9 @@ function jaeger_patch_values() {
   sed -i -e '/service:$/,/externalPort:/ {
     s/name:.*$/name: zipkin/
 	}' -e '/jaeger:$/,/^[^ 	]/ {
+          /jaeger:/a\
+  # include elasticsearch to support default configurations\
+  elasticsearch: {}
```
Comment (dgn) [inline tmp/build/patch-jaeger.sh:10]: Indentation is looking funny here, looking into that right now

**Comment by knrc** [comment]
This is now ready to merge on second approval

**Review by dgn — COMMENTED** [review]
looking into indentation

**Comment by dgn** [comment]
Nevermind, indentation is correct (in the YAML that is produced, that is)

File: tmp/build/patch-jaeger.sh (hunk 1 above)
Comment (knrc) [reply tmp/build/patch-jaeger.sh:10]: I think there is a mixture of tabs and spaces within that file, I didn't tidy that up as part of this commit.

**Review by jwendell — APPROVED** [review]

**Pull request merged**